### PR TITLE
a few updates for the foundation site. 

### PR DESCRIPTION
**Pull request by MasonEgger**
Adding Texas Python Community more for SEO purposes

### CODE REVIEW

File: foundation/docs/foundation/about.md [hunk 1]
```diff
@@ -5,4 +5,16 @@ title: About Us
...
 ---
 # About Us
-The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer group. We are Python enthusiasts that want to share the programming language with the world, starting right here in Texas. The PyTexas Conference is currently the foundation’s only fundraising event of the year.
\ No newline at end of file
+The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer 
+group. We are Python enthusiasts that want to share the programming language 
+with the world, starting right here in Texas. We host and support various programs
+that benefit the Python Community in Texas. We host an annual conference and 
+monthly meetup, maintain a virtual community for the Python developers in Texas
+and beyond, and support various meetups in cities throughout the state.
```
Comment (toonarmycaptain) [inline foundation/docs/foundation/about.md]: Maybe change `various programs` to `a variety of` or eliminate the repeated `various` in `support various meetups` to `support meetups`

Comment (MasonEgger) [reply foundation/docs/foundation/about.md]: Done

File: foundation/docs/foundation/about.md [hunk 2]
```diff
@@ -5,4 +5,16 @@ title: About Us
 summary: Who we are
 ---
 # About Us
-The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer group. We are Python enthusiasts that want to share the programming language with the world, starting right here in Texas. The PyTexas Conference is currently the foundation’s only fundraising event of the year.
\ No newline at end of file
+The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer
```
Comment (nimbinatus) [inline foundation/docs/foundation/about.md:8]: Any specific reason to wrap other than aesthetics?

File: foundation/docs/foundation/about.md [hunk 3]
```diff
@@ -5,4 +5,16 @@ title: About Us
...
 ---
 # About Us
-The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer group. We are Python enthusiasts that want to share the programming language with the world, starting right here in Texas. The PyTexas Conference is currently the foundation’s only fundraising event of the year.
\ No newline at end of file
+The PyTexas Foundation is a 501(c)3 non-profit run by a Texas-based volunteer 
+group. We are Python enthusiasts that want to share the programming language 
+with the world, starting right here in Texas. We host and support a variety of 
+programs that benefit the Python Community in Texas. We host an annual conference 
+and monthly meetup, maintain a virtual community for the Python developers in Texas
+and beyond, and support various meetups in cities throughout the state.
```
Comment (nimbinatus) [inline foundation/docs/foundation/about.md]:
```suggestion
programs that benefit the Python community in Texas, including an annual conference 
and virtual monthly meetup, a virtual community for the Python developers in Texas
and beyond, and various in-person meetups in cities throughout the state.
```

File: foundation/docs/foundation/about.md [hunk 4]
```diff
@@ -5,4 +5,16 @@ title: About Us
...
+
+## History
+Founded in 2014, meetup organizers from Houston, Austin, and San Antonio came
+together to establish a non-profit organization who's mission would be to
```
Comment (nimbinatus) [inline foundation/docs/foundation/about.md]:
```suggestion
together to establish a non-profit organization whose mission would be to 
```

File: foundation/docs/foundation/about.md [hunk 5]
```diff
@@ -5,4 +5,16 @@ title: About Us
...
+group. We are Python enthusiasts that want to share the programming language 
+with the world, starting right here in Texas. We host and support a variety of 
+programs that benefit the Python Community in Texas. We host an annual conference 
+and monthly meetup, maintain a virtual community for the Python developers in Texas
+and beyond, and support various meetups in cities throughout the state.
+
+## History
+Founded in 2014, meetup organizers from Houston, Austin, and San Antonio came
+together to establish a non-profit organization who's mission would be to 
+serve the Python Community In Texas. Glen Zangirolami, Jeremy Boyd, and Kevin
```
Comment (nimbinatus) [inline foundation/docs/foundation/about.md]:
```suggestion
serve the Python community in Texas. Glen Zangirolami, Jeremy Boyd, and Kevin
```

File: foundation/docs/foundation/about.md [hunk 6]
```diff
@@ -5,4 +5,16 @@ title: About Us
...
+Horn were the Foundation's founders and first Board of Directors. In 2022 Mason
```
Comment (nimbinatus) [inline foundation/docs/foundation/about.md]:
```suggestion
Horn were the Foundation's founders and first Board of Directors. In 2022, Mason
```

File: foundation/docs/index.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: The PyTexas Foundation
 template: home.html
 ---
 
-# Welcome to the PyTexas Foundation
+# Supporting the Python Community in the great state of Texas
```
Comment (nimbinatus) [inline foundation/docs/index.md]:
```suggestion
# Supporting the Python community in Texas
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]

**Pull request merged**